templates/prelogin/base.html

```diff
@@ -205,6 +205,32 @@ <h3 class="modal-title">
                 </div>
             </div>
         </div>
+        <div id="contactUsFormErrors"
+             class="modal fade">
+            <div class="modal-dialog">
+                <div class="modal-content">
+                    <div class="modal-header">
+                        <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">{% trans 'Close' %}</span></button>
+                        <h3 class="modal-title">
+                            {% trans "Ooops!" %}
+                        </h3>
+                    </div>
+                    <div class="modal-body">
+                        <p>
+                            {% comment %}
+                                There is only one field that can result in errors,
+                                so it's safe to assume the problem is the e-mail
+                                address.
```

Comment:
I was thinking about this over my lunch today. I agree that we ought to use the errors dictionary we get from Django. At the moment, only the e-mail address field has any requirements (must be populated, and must pass [Django e-mail validation](https://github.com/django/django/blob/master/django/core/validators.py#L137)). But using the errors dictionary would make this a bit more future-resistant, and simpler to extend.